src/renderer/ssf-api.ts

```diff
@@ -471,11 +472,9 @@ export class SSFApi {
    * Opens a modal window to configure notification preference.
    */
   public showNotificationSettings(data: string): void {
-    const windowName = (remote.getCurrentWindow() as ICustomBrowserWindow)
-      .winName;
     local.ipcRenderer.send(apiName.symphonyApi, {
       cmd: apiCmds.showNotificationSettings,
-      windowName,
+      windowName: 'main',
```

Comment:
It might be worth to add a const MAIN_WINDOW_NAME = 'main' and then use it here.